### PR TITLE
fix url parsing for codeforces problemset

### PR DESCRIPTION
**Pull request by ffao**
In problemset, the number comes after problem (i.e. http://codeforces.com/problemset/problem/529/B), so original regex was failing for this case.

### CODE REVIEW

**Comment by EgorKulikov** [comment]
What's the point of double ^ at the start?

File: extension/background.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ chrome.tabs.onUpdated.addListener(checkForValidUrl);
 function parseTask(tab) {
     if (/^https:\/\/.*contest2?[.]yandex[.](ru|com)\/.*contest\/\d*\/problems.*$/.test(tab.url)) {
         chrome.tabs.sendMessage(tab.id, 'yandex');
-    } else if (/^http:\/\/codeforces[.](ru|com)\/(contest|problemset|gym)\/\d*\/problem\/.+$/.test(tab.url)) {
+    } else if (/^^http:\/\/codeforces[.](ru|com)\/(contest|problemset|gym)\/(\d*\/problem|problem\/\d*)\/.+$/.test(tab.url)) {
```
Comment (EgorKulikov) [inline extension/background.js]: Why double ^ ?

Comment (ffao) [inline extension/background.js]: I must have checked the diff at least 3 times and I missed that, removed it.

**Comment by EgorKulikov** [comment]
Lgtm

**Pull request merged**